integration/cypress/integration/accessibility/accessibility.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { generateLegacyURL, generateNewURL } from "@maas-ui/maas-ui-shared";
+
+const legacyPages = ["/devices", "/controllers", "/networks"];
```

Comment:
We can move `/devices` out of legacy pages now that we're using the React version